node/service/src/lib.rs

```diff
@@ -593,9 +593,9 @@ where
 				&task_manager.spawn_essential_handle(),
 				config.prometheus_registry(),
 				if experimental_block_import_strategy {
-					None
+					true
 				} else {
-					Some(!dev_service)
+					!dev_service
```

Comment:
Should be set to false:
```suggestion
					false
```